boards/apollo3/lora_things_plus/src/tests/verify_sig.rs

```diff
@@ -0,0 +1,171 @@
+// Licensed under the Apache License, Version 2.0 or the MIT License.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+// Copyright Tock Contributors 2022.
+
+use crate::tests::run_kernel_op;
+use crate::ATECC508A;
+use core::cell::Cell;
+use kernel::hil::public_key_crypto::signature::ClientVerify;
+use kernel::hil::public_key_crypto::signature::SignatureVerify;
+use kernel::static_init;
+use kernel::utilities::cells::TakeCell;
+use kernel::{debug, ErrorCode};
+
+struct HmacTestCallback {
+    verify_done: Cell<bool>,
+    message_buffer: TakeCell<'static, [u8; 32]>,
+    signature_buffer: TakeCell<'static, [u8; 64]>,
+    pub_key_buffer: TakeCell<'static, [u8; 64]>,
+}
+
+unsafe impl Sync for HmacTestCallback {}
```

Comment:
I realize this is needed in some other tests, but this test builds fine without it from what I can tell and I would prefer not to have this pattern proliferate. Can we remove this?

Reply:
Fixed